main/build.gradle

```diff
@@ -222,6 +222,7 @@ android {
         }
         // debug and release builds have offset zero, no special handling here
     }
+    buildToolsVersion '25.0.0'
```

Comment:
Why are you doing that? The default is `25.0.2` right now, and is enforced near the beginning of this file by getting it from the upper `build.gradle` file.